docs/components/table.md

```diff
@@ -139,37 +149,25 @@ export default {
   }
 }
 </script>
-<style>
-.clickable-rows tr {
-  cursor: pointer;
-}
-</style>
 ```
```

Comment:
can you add vue to this code block?
![image](https://user-images.githubusercontent.com/5941111/86197564-81cd3d80-bb0a-11ea-9d52-9a32fb05681f.png)